oaknational/Oak-Web-Application
# feat: self host mathjax - LESQ-386

## Description

- Host mathjax from public folder
- Update provider 

## Acceptance Criteria:

- [ ]  MathsJax is still able to render maths notation correctly
- [ ]  MathsJax is self hosted and not sourced from a CDN

## Issue(s)



## How to test

1. Go to {owa_deployment_url}/teachers/programmes/maths-secondary-ks4-foundation/units/surds/lessons/accuracy-of-final-answers#exit-quiz
You should see mathjax in the quizes



## Checklist
